03-file-operations/CreateFile.java

```diff
@@ -0,0 +1,38 @@
+import java.io.File;
+
+/**
+ * CreateFile Class.
+ */
+public class CreateFile {
+
+    /**
+     * Main Function.
+     *
+     * @param String[] args
+     */
+    public static void main( String[] args ) {
+
+        // Create file object.
+        File file = new File( "src/programming-in-java/03-file-operations/new-file.txt" );
+
+        try {
+
+            if ( file.createNewFile() ) {
+                System.out.println( "New file is created" );
+            } else {
+                System.out.println( "File already exists" );
+            }
+
+            // Print absolute path.
+            System.out.println( file.getAbsolutePath() );
+
+            // Check if the file is readable and writable.
+            System.out.println( file.canRead() );
```

Comment:
Add `if` statement to print if the file is readable/writable. @imranhsayed

Reply:
I think this is fin @sumayyahc1 . This will just print the result if it's readable or not.